wiggle_sort.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+
+class Myclass:
+	"""
+	Given an unsorted array nums, 
+	reorder it in-place such that nums[0] <= nums[1] >= nums[2] <= nums[3]....
+	For example, given nums = [3, 5, 2, 1, 6, 4],
+	one possible answer is [1, 6, 2, 5, 3, 4].
+	"""
+	def wiggle_sort(self, A):
+		"""
+		:type A:[int]
+		:rtype: void
+		"""
+		for i in range(1, len(A)):
+			if i % 2 and A[i] < A[i - 1]:
+				A[i], A[i - 1] = A[i - 1], A[i]
```

Comment:
line 18 == line 20, maybe you can merge the conditions:
if (i % 2 and A[i] < A[i - 1]) or (i % 2 == 0 and A[i] > A[i - 1]):